### DISCUSSION TITLE
Not able install with .Net Standard 2.0/2.1

### DISCUSSION THREAD

**Discussion by jojopeter** [dotnet/yarp#863] [Q&A] [answered]
Hi,
I am trying to install the package on a .NET Standard 2.0 class library and it's compatible. Even I tried to install in a library with framework .NET Standard 2.1 and unsuccessful. Why even .Net standard 2.1 is not supported?

I am getting below error. 

_Install-Package : NU1202: Package Microsoft.ReverseProxy 1.0.0-preview.7.20562.2 is not compatible with netstandard2.1 (.NETStandard,Version=v2.1). Package Microsoft.ReverseProxy 1.0.0-preview.7.20562.2 supports:
  - net5.0 (.NETCoreApp,Version=v5.0)
  - netcoreapp3.1 (.NETCoreApp,Version=v3.1)_
  - 

Any help greatly appreciated. Thanks.

**Comment by Tratcher** [comment, marked as answer]
This library directly integrates with ASP.NET Core 3.1+. What are you trying to use it for in a net standard library?
